src/utils/getSketchVersion.ts

```diff
@@ -1,6 +1,5 @@
-export function getSketchVersion(): number | 'NodeJS' {
-  if (typeof NSBundle !== 'undefined') {
-    return parseFloat(NSBundle.mainBundle().infoDictionary().CFBundleShortVersionString);
-  }
-  return 'NodeJS';
+export default function getSketchVersion(): string {
```

Comment:
any reason for this change?

Reply:
One thing that occurred to me is that with the bridge system, this package could be running also in the browser, so returning `"NodeJS"` would be misleading. 

Also, parsing the date as a float, in general, is discouraged: in case there are patch versions, you lose that information; but more importantly, the direct naive comparison of floats would bring the wrong results: v1.10 > v1.9, but 1.10 < 1.9, so naive code would do the wrong thing.